tutorials/R/Geospatial-skills/intro-vector-r/00-open-a-shapefile/00-open-a-shapefile.Rmd

```diff
@@ -250,7 +250,7 @@ attributes stored with it.
 
 <figure>
     <a href="https://raw.githubusercontent.com/NEONScience/NEON-Data-Skills/dev-aten/graphics/dc-spatial-vector/Attribute_Table.png">
-    <img src="https://raw.githubusercontent.com/NEONScience/NEON-Data-Skills/dev-aten/graphics/dc-spatial-vector/Attribute_Table.png"></a>
+    <img src="https://raw.githubusercontent.com/NEONScience/NEON-Data-Skills/dev-aten/graphics/dc-spatial-vector/Attribute_Table.png"> alt="A figure depicting example attribute tables for each different type of vector object."</a>
```

Comment:
There's an old motto in the education literature that, to remember any new skill, people have to practice it three times.. :)